### DISCUSSION TITLE
Execution scala-compile-first of goal net.alchim31.maven:scala-maven-plugin:3.2.1:compile failed: invalid versionNumber : major.minor

### DISCUSSION THREAD

**Discussion by Cyb315966632** [apache/doris#13686] [RFC]
Execution scala-compile-first of goal net.alchim31.maven:scala-maven-plugin:3.2.1:compile failed: invalid versionNumber : major.minor

**Comment by hf200012** [comment]
Send out pom.xml to have a look
Did you compile the flink connector or what?
